src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/EventTypeAT.java

```diff
@@ -0,0 +1,170 @@
+package de.zalando.aruha.nakadi.webservice;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.config.NakadiConfig;
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.EventTypeSchema;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+
+public class EventTypeAT extends BaseAT {
+
+    static private final String ENDPOINT = "/event_types";
+    private final ObjectMapper mapper = (new NakadiConfig()).jacksonObjectMapper();
+
+    @Test
+    public void whenGETThenListsEventTypes() throws JsonProcessingException {
```

Comment:
Ok, I actually thought that for user journey we'll have separate test. And the acceptance tests are more like API-tests for me, where we test Nakadi functionality through it's API.

But yes, let's discuss it on our meeting. (for me the current implementation is quite fine)